bears/vcs/git/GitCommitBear.py

```diff
@@ -193,25 +209,19 @@ def check_imperative(self, paragraph):
             A list of tuples having 2 elements (invalid word, parts of speech)
             or an empty list if no invalid words are found.
         """
-        try:
-            words = nltk.word_tokenize(nltk.sent_tokenize(paragraph)[0])
-            # VBZ : Verb, 3rd person singular present, like 'adds', 'writes'
-            #       etc
-            # VBD : Verb, Past tense , like 'added', 'wrote' etc
-            # VBG : Verb, Present participle, like 'adding', 'writing'
-            word, tag = nltk.pos_tag(['I'] + words)[1:2][0]
-            if(tag.startswith('VBZ') or
-               tag.startswith('VBD') or
-               tag.startswith('VBG') or
-               word.endswith('ing')):  # Handle special case for VBG
-                return (word, tag)
-            else:
-                return None
-        except LookupError as error:  # pragma: no cover
-            self.err('NLTK data missing, install by running following '
-                     'commands `python3 -m nltk.downloader punkt'
-                     ' maxent_treebank_pos_tagger averaged_perceptron_tagger`')
-            return
+        words = nltk.word_tokenize(nltk.sent_tokenize(paragraph)[0])
+        # VBZ : Verb, 3rd person singular present, like 'adds', 'writes'
+        #       etc
+        # VBD : Verb, Past tense , like 'added', 'wrote' etc
+        # VBG : Verb, Present participle, like 'adding', 'writing'
+        word, tag = nltk.pos_tag(['I'] + words)[1:2][0]
+        if(tag.startswith('VBZ') or
+           tag.startswith('VBD') or
+           tag.startswith('VBG') or
```

Comment:
Not in this commit.  git has a special mode for indentation only changes, so that `git blame` can skip them to find real changes.